package.json

```diff
@@ -8,6 +8,9 @@
     "server-static-prod": "ecstatic --port 8080 --baseDir /open-source/victory build",
     "start": "builder run dev",
     "test": "builder run lint",
+    "test-func-install": "builder run install-selenium",
+    "test-func-build": "NODE_ENV=production builder run build-static",
+    "test-func": "ROWDY_SETTINGS=local.chrome builder run test-func-static-prod",
```

Comment:
While travis doesn't have to be portable (because it's targeted to linux), our `package.json:scripts` should be, especially when it's super easy to make this both Win + Mac compatible with:

```json
"test-func-build": "builder run --env='{ \"NODE_ENV\": \"production\" }' build-static",
"test-func": "builder run --env='{ \"ROWDY_SETTINGS\": \"local.chrome\" }' test-func-static-prod",
```